test/models/activation_key_test.rb

```diff
@@ -8,6 +8,7 @@ def setup
       @dev_view = katello_content_views(:library_dev_view)
       @lib_view = katello_content_views(:library_view)
       @pool_one = katello_pools(:pool_one)
+      @purpose_attributes = katello_activation_keys(:key_purpose_attributes)
```

Comment:
maybe 'purpose_key' would be a better name for this variable ?

Reply:
Yah. That sounds good. Renamed it. :+1: